README.md

```diff
@@ -0,0 +1,34 @@
+# Flathub: Crossfire rpg
```

Comment:
There's generally no need to have any README here, as it will not be seen by any user.

Reply:
OK but since I put one together, it might as well be there.